src/core.js

```diff
@@ -284,6 +284,16 @@ function done() {
 		total: config.stats.all,
 		runtime: runtime
 	} );
+
+	// Clear own sessionStorage items if all tests passed
+	if ( config.reorder && defined.sessionStorage && config.stats.bad === 0 ) {
+		for ( i = 0; i < sessionStorage.length; i++ ) {
+			key = sessionStorage.key( i++ );
```

Comment:
Since that bug was present before, I wonder if the whole implementation is flawed. Removing items from an object while iterating over it also seems fishy.